EngineLayer/Indexing/IndexingEngine.cs

```diff
@@ -233,7 +234,7 @@ private List<int>[] CreateNewPrecursorIndex(List<PeptideWithSetModifications> pe
                     }
                 }
                 progress++;
-                var percentProgress = (int)((progress / maxProgress) * 100);
```

Comment:
This line should be "int percentProgress = (int)((progress*1d/maxProgress)*100);"
In its current form, the percentProgress can only ever be 0 or 1.